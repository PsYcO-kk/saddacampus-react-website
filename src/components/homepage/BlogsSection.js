import React from 'react';
import Blog from './Blog';

const BlogsSection  = () => (
    <section className="latest-blog-area section-gap">
		<div className="container">
			<div className="row section-title">
				<h1>Latest News from our Blog</h1>
				<p>Lorem ipsum dolor sit amet</p>
			</div>
			<div className="row">
				<Blog imgSrc="coffee.PNG" />
				<Blog imgSrc="burger.PNG" />
			</div>
		</div>
	</section>
);

export default BlogsSection;
