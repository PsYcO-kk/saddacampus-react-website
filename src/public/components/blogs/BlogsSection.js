import React from 'react';
import { Row, Col } from 'reactstrap';

import BlogCard from './BlogCard';

const BlogsSection = (props) => (
	<div className="blogs-section">
		<h3>FEATURED</h3>
		{
			props.type === "featured" ?
				(
					<Row className="remove-extra-space">
						<Col md={7} className="remove-extra-space">
							<BlogCard type="large" />
						</Col>
						<Col md={5} className="remove-extra-space">
							<BlogCard type="small" />
							<BlogCard type="small" />
						</Col>
					</Row>
				) :
				(
					<Row className="remove-extra-space">
						<Col md={6} className="remove-extra-space">
							<BlogCard type="medium" />
						</Col>
						<Col md={6} className="remove-extra-space">
							<BlogCard type="medium" />
						</Col>
					</Row>
				)
		}
		<div style={{ textAlign: 'right', paddingTop: '2%', marginRight: '2%' }}>
			View More
		</div>
	</div>
);

export default BlogsSection;
