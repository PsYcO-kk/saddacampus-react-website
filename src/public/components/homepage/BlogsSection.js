import React from 'react';
import { Row, Col, Button } from 'reactstrap';

const BlogsSection  = () => (
    <section className="section-gap-half">
		<Row className="remove-extra-space align-items-center">
			<Col md={5} className="remove-extra-space">
				<div style={{ margin: '0 20%' }}>
					<div style={{ textAlign: 'right' }}>
						<h1 style={{ fontFamily: 'Lato, Black' }}>Latest News from our Blog</h1>
						<p style={{ fontFamily: 'Lato, Regular', fontSize: '20px', lineHeight: '23px', margin: '7% 0' }}>There are always some things you need to know and once in college you need all the help available.</p>
						<Button outline color="primary" style={{ borderRadius: 0 }}>VIEW BLOG</Button>
					</div>
				</div>
			</Col>
			<Col md={7} className="remove-extra-space">
				<div className="thumb" style={{ textAlign: 'right', margin: '2% 0' }}>
					<img className="img-fluid" src="/assets/images/beauty-black-and-white-color-splash-46171.jpg" alt="" />
				</div>
			</Col>
		</Row>
	</section>
);

export default BlogsSection;
