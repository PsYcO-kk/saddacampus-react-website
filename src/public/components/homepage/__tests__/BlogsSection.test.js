import {shallow} from 'enzyme';
import React from 'react';
import BlogsSection from '../BlogsSection';

test("should render BlogsSection correctly" , ()=>{
    const wrapper = shallow(<BlogsSection />)
    expect(wrapper).toMatchSnapshot();
})
