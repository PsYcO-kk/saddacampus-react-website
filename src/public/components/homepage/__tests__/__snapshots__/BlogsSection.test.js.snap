// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BlogsSection correctly 1`] = `
<section
  className="latest-blog-area section-gap"
>
  <div
    className="container"
  >
    <div
      className="row section-title"
    >
      <h1>
        Latest News from our Blog
      </h1>
      <p>
        Lorem ipsum dolor sit amet
      </p>
    </div>
    <div
      className="row"
    >
      <Blog
        imgSrc="coffee.PNG"
      />
      <Blog
        imgSrc="burger.PNG"
      />
    </div>
  </div>
</section>
`;
